studio.config.json

```diff
@@ -3,43 +3,11 @@
     "dir": "",
     "menu": [
       "introduction",
-      [
-        "TailwindCSS",
-        [
-          "tailwind-showcases"
-        ]
-      ],
-      [
-        "CSS",
-        [
-          "css-showcases"
-        ]
-      ],
-      [
-        "Doc-site",
-        [
-          "layout",
-          "speedy-links"
-        ]
-      ],
-      [
-        "Components",
-        [
-          "box",
-          "clipboard",
-          "text",
-          "showcases",
-          "space",
-          "do-dont",
-          "search"
-        ]
-      ],
-      [
-        "Infra",
-        [
-          "doc-layout"
-        ]
-      ]
+      ["DESIGN TOKEN SHOWCASES", ["tailwind-showcases", "css-showcases"]],
```

Comment:
I think capital letters is a presentational layer, it comes from CSS, so I don't get why you changed it here?